### DISCUSSION TITLE
nice video

### DISCUSSION THREAD

**Discussion by stsp** [dosemu2/dosemu2#1369] [General]
https://www.youtube.com/watch?v=sM5tamVvDwQ

Makes me feel sorry we do not support
these old systems. :)

**Comment by stuaxo** [comment]
I like the casual way he just has DOS networking up on it too :)

**Comment by stsp** [comment]
Never thought older dosemu needed
a DOS boot diskette to start. I suppose
this is when it didn't yet have fatfs
emulation. But no problem, he just
inserted the boot diskette, and it all
booted up!

Lets try a newer one, with fatfs.
Oh, segfault... No problem, I guess
I just compile from sources.
Shit, the sources do not compile...
But not a problem, I can fix here,
fix there, and (3 hours later) it works! :)

This guy is really cool.
A good reminder of how difficult it
was to run dosemu1. :)
